### PR TITLE
Sort imports

### PR DESCRIPTION
**Pull request by ostatni5**

### CODE REVIEW

**Comment by grzanka** [comment]
@ostatni5 will this work with husky ?

**Review by Derstilon — CHANGES_REQUESTED** [review]
Update lint-staged config in package json to include autofix on comitted files

File: package.json [hunk 1]
```diff
@@ -88,6 +88,9 @@
 			"react-app",
 			"react-app/jest"
 		],
+		"plugins": [
+			"sort-imports-es6-autofix"
```
[the remark this answers is not in the document]
Comment (ostatni5) [reply package.json]: Seems legit

**Comment by grzanka** [comment]
image

tests are failing

**Review by Derstilon — APPROVED** [review]

**Pull request merged**